src/ape/api/accounts.py

```diff
@@ -93,10 +93,16 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
         txn = self.prepare_transaction(txn)
 
         if send_everything:
+            error_message = "Sender does not have enough to cover transaction value and gas: "
             if txn.max_fee is None:
                 raise TransactionError(message="Max fee must not be None.")
```

Comment:
Can we raise a `ValueError` if both `txn.value` and `send_everything=True` is specified?

Reply:
sure, that seems like a good idea

Reply:
implemented